README.md

```diff
@@ -61,8 +60,28 @@ Note: If you have cloned the repo previously make sure you remove your node_modu
 - `./gradlew publishAndroid`
 - The compiled version of the Android module is here: `<project-root>/android`
 
+Note: On Windows the RealmJS repo should be cloned with symlynks enabled 
```

Comment:
Shouldn't this be mentioned further up under the "Clone RealmJS repository:" section?

Reply:
yes. will fix